### PR TITLE
No Frame Login Screen

### PR DESCRIPTION
**Pull request by tmccormi**
This is a no Frame version of the login screen.  Screen shot included

noframelogin

### CODE REVIEW

File: admin.php [hunk 1]
```diff
@@ -13,8 +13,8 @@
   $webserver_root = str_replace("\\","/",$webserver_root);
```
Comment (bradymiller) [inline admin.php]: changes in this file should be reverted

File: interface/login/login.php [hunk 2]
```diff
@@ -129,6 +128,7 @@ function imsubmitted() {
 <table width="100%" height="90%">
 <td align='center' valign='middle' width='34%'>
 <div class="login-box">
+<img src="/emr_devtip/openemr/interface/pic/logo.gif" />
```
Comment (bradymiller) [inline interface/login/login.php]: this appears like something that is hard-coded and will break

**Comment by bradymiller** [comment]
Wouldn't it make more sense to consolidate the following files and remove the frames:
login_frame.php
login_title.php
filler.php
login.php

**Comment by bradymiller** [comment]
review complete

**Comment by tmccormi** [comment]
Yep, all true.  This is intended as an example really, just sharing what we did.

**Pull request closed without merging**